routing/src/lib.rs

```diff
@@ -11,7 +11,7 @@ mod encapsulation;
 mod errors;
 mod interface;
 mod nexthop;
-mod prefix;
+pub mod prefix;
```

Comment:
Yes all of those will need to be public. I have a local commit. I can open PR  with some fixes like that.

Reply:
Right now I only needed to access the `Prefix` stuff, so no rush from my side. But sure, happy to review a PR once you have something ready